driver/pelican/main.go

```diff
@@ -53,6 +53,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	pollDrStr := params.MustString("poll_interval_dr")
+	pollDr, drErr := time.ParseDuration(pollDrStr)
+	if drErr != nil {
+		fmt.Printf("Invalid demand response poll interval specified: %v\n", err)
```

Comment:
Probably put `drErr` here instead of `err`

Reply:
Ahhh I keep making these mistakes, I'll be more thorough in the future, sorry about that haha.